internal/sql/bean/bean.go

```diff
@@ -42,22 +43,68 @@ type ImageScanOutputObject struct {
 
 type Severity int
 
+const (
+	HIGH     string = "high"
+	CRITICAL string = "critical"
+	SAFE     string = "safe"
+	LOW      string = "low"
+	MEDIUM   string = "medium"
+	MODERATE string = "moderate"
+)
+
 const (
 	Low Severity = iota
 	Medium
-	High
 	Critical
+	High
+	Safe
 )
 
 func (sev Severity) String() string {
-	return [...]string{"Low", "Medium", "High", "Critical"}[sev]
+	return [...]string{"low", "medium", "critical", "high", "safe"}[sev]
+}
+func ConvertToLowerCase(input string) string {
+	return strings.ToLower(input)
 }
 
 var ConvertToSeverity = map[string]Severity{
-	"Low":      Low,
-	"Medium":   Medium,
-	"High":     High,
-	"Critical": Critical,
+	"low":      Low,
+	"medium":   Medium,
+	"high":     Critical,
+	"critical": Critical,
+	"safe":     Low,
+}
+var ConvertToStandardSeverity = map[string]Severity{
+	"low":      Low,
+	"medium":   Medium,
+	"high":     High,
+	"critical": Critical,
+	"safe":     Safe,
+}
+
+func ConvertToSeverityUtility(severity string) Severity {
+	if severity == LOW || severity == SAFE {
+		return Low
+	} else if severity == MEDIUM {
+		return Medium
+	} else if severity == HIGH || severity == CRITICAL {
+		return Critical
+	}
+	return Low
+}
+func ConvertToStandardSeverityUtility(severity string) Severity {
+	if severity == LOW {
+		return Low
+	} else if severity == MEDIUM {
```

Comment:
handling for moderate?

Reply:
moderate severity do not come for any tool clair v4 or clair v2, so no handling is required.